diff --git a/node_modules/@react-native-community/datetimepicker/src/datetimepicker.android.js b/node_modules/@react-native-community/datetimepicker/src/datetimepicker.android.js
index 1111111..2222222 100644
--- a/node_modules/@react-native-community/datetimepicker/src/datetimepicker.android.js
+++ b/node_modules/@react-native-community/datetimepicker/src/datetimepicker.android.js
@@ -2,6 +2,7 @@
  * @format
  * @flow strict-local
  */
+// @ts-nocheck
 import {ANDROID_DISPLAY, ANDROID_MODE} from './constants';
 import {useEffect} from 'react';
 
